pkg/certsuite/certsuite.go

```diff
@@ -50,7 +50,7 @@ func LoadChecksDB(labelsExpr string) {
 }
 
 const (
-	junitXMLOutputFileName = "cnf-certification-tests_junit.xml"
+	junitXMLOutputFileName = "certsuite-tests_junit.xml"
```

Comment:
This can't change without DCI also making a change at the same time as they are looking for this XML file.

cc @ramperher

Reply:
Can we please make DCI changes for this renaming, @ramperher?

Reply:
done